src/instance.js

```diff
@@ -976,12 +992,21 @@ Instance.prototype.setMetadata = function(metadata, gaxOptions, callback) {
     callback = gaxOptions;
     gaxOptions = {};
   }
+  var updateMask = metadata.updateMask
```

Comment:
Only set items that exist on the metadata object. Follow the same pattern as https://github.com/googleapis/nodejs-bigtable/blob/c2ce475afa903939419fb708837b4692a7df0b76/src/app-profile.js#L364-L375